### PR TITLE
Display warning overlay instead of scene when below min spec

### PR DESCRIPTION
**Pull request by jherico**
When launched from Steam, if the machine is below minimum spec, this will display an image in an overlay within the HMD, rather than the scene.  

## Testing

Launch from steam on an i5 based machine.  Instead of a scene in the HMD, all you should see is a warning overlay.

### CODE REVIEW

File: plugins/openvr/src/OpenVrDisplayPlugin.cpp [hunk 1]
```diff
@@ -39,10 +39,6 @@ const QString StandingHMDSensorMode = "Standing HMD Sensor Mode"; // this probab
 PoseData _nextRenderPoseData;
 PoseData _nextSimPoseData;
 
-#define MIN_CORES_FOR_NORMAL_RENDER 5
```
Comment (ZappoMan) [inline plugins/openvr/src/OpenVrDisplayPlugin.cpp]: don't remove that...
we still want people to be able to run on an i5... just not from steam launch

File: plugins/openvr/src/OpenVrDisplayPlugin.cpp [hunk 2]
```diff
@@ -411,12 +412,6 @@ bool OpenVrDisplayPlugin::internalActivate() {
         _cullingProjection = _eyeProjections[0];
     });
 
-    // enable async time warp
-    if (forceInterleavedReprojection) {
```
Comment (ZappoMan) [inline plugins/openvr/src/OpenVrDisplayPlugin.cpp]: keep this

File: plugins/openvr/src/OpenVrDisplayPlugin.cpp [hunk 3]
```diff
@@ -39,11 +39,11 @@ const QString StandingHMDSensorMode = "Standing HMD Sensor Mode"; // this probab
 PoseData _nextRenderPoseData;
 PoseData _nextSimPoseData;
 
-#define MIN_CORES_FOR_NORMAL_RENDER 5
+static std::array<vr::Hmd_Eye, 2> VR_EYES {#define MIN_CORES_FOR_NORMAL_RENDER 5
```
Comment (ZappoMan) [inline plugins/openvr/src/OpenVrDisplayPlugin.cpp]: something still jacked here

**Comment by hifi-gustavo** [comment]
Build console output for commit: e059a0b26cb6eda97c0ad313618eafb8ec5d5779 available at: https://gist.github.com/bcc0b2c1b76578d887aaa1642f3b373f<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8921-e059a0b.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8921-e059a0b.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 30075b063b61bd771b8566fab7446e14b5e9f409 available at: https://gist.github.com/bcc0b2c1b76578d887aaa1642f3b373f<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8921-30075b0.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8921-30075b0.dmg

**Comment by jherico** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 30075b063b61bd771b8566fab7446e14b5e9f409 available at: https://gist.github.com/bcc0b2c1b76578d887aaa1642f3b373f<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8921-30075b0.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8921-30075b0.dmg

**Comment by howard-stearns** [comment]
The .png displays in GitHub as a cryptic Windows BSOD. Is that intentional?

**Comment by jherico** [comment]
@howard-stearns it's a placeholder image

**Comment by hifi-gustavo** [comment]
Build console output for commit: 2b25cb335341284142ca0044175081d7b6153661 available at: https://gist.github.com/bcc0b2c1b76578d887aaa1642f3b373f<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8921-2b25cb3.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8921-2b25cb3.dmg

**Comment by ZappoMan** [comment]
I might need to retest this... but this seemed to fail when I tested it on my AMD+i5 -- basically interface just hung... and I could see what appeared to be the first frame of rendering the scene.

**Comment by ZappoMan** [comment]
I don't know how to test this... I have an i5+AMD - even running Steam Beta, this should be below spec. I tried to use @Atlante45 's trick of setting an appid so it would think it was launched from steam, but I don't see the warning. @jherico @Atlante45 - how do we test this??

**Comment by RyanDowne** [comment]
@jherico we are going to replace the placeholder today. Is there a spec for the image size and type we should use?

**Comment by ZappoMan** [comment]
I am digging deeper into this startup problem. It seems to related to the code to determine if we're running from steam.

**Comment by ZappoMan** [comment]
This PR will need to have this fix merged before it can be tested... https://github.com/highfidelity/hifi/pull/8930

**Comment by jherico** [comment]
@RyanDowne the image should be <= 1920x1080

**Comment by ZappoMan** [comment]
testing again

**Comment by ZappoMan** [comment]
crashed on i5+AMD+SteamVR (non-beta)

**Comment by hifi-gustavo** [comment]
Build console output for commit: 994d420154b29835e2f52744a8a850c5dbcfa28f available at: https://gist.github.com/bcc0b2c1b76578d887aaa1642f3b373f<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8921-994d420.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8921-994d420.dmg

**Comment by ZappoMan** [comment]
Tested on i5/AMD/Steam(non-beta): FAIL
- does render the "BSOD" image - pass
- image is tilted sideways and kind of rotated
- no way to exit
- system becomes nearly unresponsive and I had to use task manager to kill interface
- couldn't bring up steam menu to exit...

**Comment by hifi-gustavo** [comment]
Build console output for commit: 72a22a3d7ed7c80f125dec6aafc4bacbc5dbe4bf available at: https://gist.github.com/bcc0b2c1b76578d887aaa1642f3b373f<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8921-72a22a3.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8921-72a22a3.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 84dde7e8043a67c172333977464e5b5dc4112309 available at: https://gist.github.com/bcc0b2c1b76578d887aaa1642f3b373f<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8921-84dde7e.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8921-84dde7e.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 018d945b85b258ecb2d62295e259113518f0b9f7 available at: https://gist.github.com/bcc0b2c1b76578d887aaa1642f3b373f<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8921-018d945.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8921-018d945.dmg

**Comment by RyanDowne** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 018d945b85b258ecb2d62295e259113518f0b9f7 available at: https://gist.github.com/bcc0b2c1b76578d887aaa1642f3b373f<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8921-018d945.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8921-018d945.dmg

**Comment by ZappoMan** [comment]
testing now

**Comment by hifi-gustavo** [comment]
Build console output for commit: e9db8be43510754410af2fbb0540ac3191fb3b76 available at: https://gist.github.com/bcc0b2c1b76578d887aaa1642f3b373f<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8921-e9db8be.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8921-e9db8be.dmg

**Comment by ZappoMan** [comment]
Testing Results:
- When launched "from steam"
    - i5/AMD/SteamVR(production) -- PASS - it warned me as expected
    - i5/970/SteamVR(production) -- PASS - it warned me as expected
    - i5/970/SteamVR(beta) -- PASS - it let me use the product as expected - experience good
- When launched "stand alone"
    - i5/AMD/SteamVR(production) -- PASS - it let me run
    - i5/970/SteamVR(production) -- PASS - it let me run
    - i5/970/SteamVR(beta) -- PASS - it let me run


Feedback:
- The message text is too hard to read because the overlay is too small... we need to make the overlay bigger
- The timeout is too short, I didn't see the message while putting on my HMD

**Pull request merged**